Desktop/QueryRetrieve SCU/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
```

Comment:
Please consider re-using the *SharedAssemblyInfo.cs* file for attributes common to all applications. Also, I would appreciate it if comments could be translated into English or omitted altogether.